package_linux_bin.sh

```diff
@@ -22,6 +22,17 @@ if [[ "${VSCODE_ARCH}" == "ppc64le" ]]; then
   export VSCODE_SYSROOT_PREFIX='-glibc-2.28'
 fi
 
+if [[ "${VSCODE_ARCH}" == "riscv64" ]]; then
+  export VSCODE_ELECTRON_REPO='riscv-forks/electron-riscv-releases'
+  export ELECTRON_SKIP_BINARY_DOWNLOAD=1
+  export PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD=1
+  ELECTRON_VERSION="v$(yarn config get target)"
```

Comment:
`ELECTRON_VERSION="v29.4.0"` should be enough

Reply:
Fixed